Costume/052_Meowth.yaml

```diff
@@ -0,0 +1,21 @@
+species: Meowth
+imagename: Litten
+displayname: Litten
+setname: Costume
+tags: [costume]
+item: [Figy Berry]
+ability: [Blaze]
+moves:
+    - [Flare Blitz]
+    - [Return]
```

Comment:
I would swap return for facade, so is actually weaker than flare blitz after STAB but has psudo burn immunity like a real fire type.